integration-tests/src/test/java/org/zowe/apiml/integration/zos/PassTicketTest.java

```diff
@@ -62,8 +61,7 @@ void setUp() {
 
     protected static URI[] passTicketUrls() {
         return new URI[]{
-            HttpRequestUtils.getUriFromGateway(TICKET_ENDPOINT),
-            HttpRequestUtils.getUriFromGateway(TICKET_ENDPOINT_OLD_PATH_FORMAT)
+            HttpRequestUtils.getUriFromGateway(ROUTED_PASSTICKET)
```

Comment:
Does it make sense to have it parametrized?

Reply:
Removed